src/docs/user/references/Variables.adoc

```diff
@@ -146,7 +146,11 @@ To define a variable substitution, the workflow designer can simply use a variab
  +
 When submitting the workflow, the user will have the ability to update the models dynamically by changing the value of reference variable.  +
 
-For example: If we have two variables `A` and `B` with models `PA:LIST(PA:GLOBAL_FILE, PA:INTEGER)` and `$A`, then `$A` represents the value that the variable `A` will have in runtime. Thus, it will be either `PA:GLOBAL_FILE` or `PA:INTEGER`
+For example:  +
+`variable_name1` has model `PA:LIST(PA:GLOBAL_FILE, PA:INTEGER)` +
+`variable_name2` has model `$variable_name1` +
+
+Then the model of `variable_name2` is the value that the variable `variable_name1` will have in runtime. Thus, it will be either `PA:GLOBAL_FILE` or `PA:INTEGER`
```

Comment:
has model --> has a model or has as a model

i would select variable1 or variable1_name as name for the variable since we are talking about two different variables. The manner you did, the reader can suggest that you are talking about one single variable with two different names